tempest/api/workloadmgr/security_groups/test_security_group_remotesecuritygroup.py

```diff
@@ -0,0 +1,436 @@
+import logging
+import os
+import random
+import sys
+import time
+
+from oslo_log import log as logging
+
+from tempest import command_argument_string
+from tempest import config
+from tempest import reporting
+from tempest import test
+from tempest import tvaultconf
+from tempest.api.workloadmgr import base
+from tempest.lib import decorators
+from tempest.util import cli_parser
+from tempest.util import query_data
+
+
+sys.path.append(os.getcwd())
+
+LOG = logging.getLogger(__name__)
+CONF = config.CONF
+
+
+class WorkloadTest(base.BaseWorkloadmgrTest):
+    credentials = ["primary"]
+
+    # Delete default security group rules
+    def delete_default_rules(self, secgrp_id):
```

Comment:
move all reusable functions to base.py instead of defining them as functions here.